### PR TITLE
C sharp/atan2

### PR DESCRIPTION
**Pull request by aflondono**
### Description

Added new documentation term entry for C# `Math.Atan2()`. This required creating a new file `docs/content/c-sharp/concepts/math-functions/terms/atan2/atan2.md`.

Also edited the `tags.md` file, to add a couple of tags used in the `atan2.md` file.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/c-sharp/concepts/math-functions/terms/atan2/atan2.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Geometry'
+  - 'Methods'
+  - 'Numbers'
+  - 'Trigonometry'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'learn-c-sharp-methods'
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/atan2/atan2.md:16]:
```suggestion
```
We generally only have two entries for catalog content

File: content/c-sharp/concepts/math-functions/terms/atan2/atan2.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  - 'paths/computer-science'
+---
+
+The **Math.Atan2()** function returns the angle, in radians, between the positive x-axis and a vector to the point with the given (x, y) in the Cartesian plane.
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/atan2/atan2.md:20]:
```suggestion
The **Math.Atan2()** function returns the angle, in radians, between the positive x-axis and a vector to the point with the given (x, y) coordinates in the Cartesian plane.
```

**Review by SSwiniarski — COMMENTED** [review]
@aflondono I did a first review and only had a couple of comments.

**Comment by aflondono** [comment]
Thank you @SSwiniarski. I have committed your suggested changes.

**Review by KTom101 — APPROVED** [review]
@alfondono I made a small change here [aba64d4](https://github.com/Codecademy/docs/pull/1792/commits/aba64d46ef2e3f683560f2ffa30a4d473a9c59ea). It looks good, otherwise. 👍

**Comment by KTom101** [comment]
@aflondono Entry is up! Here is the link: https://www.codecademy.com/resources/docs/c-sharp/math-functions/atan2

Currently, you're listed as anonymous:

Screenshot 2023-03-21 at 9 36 59 PM

If you want to remain anonymous, there is nothing more you'll need to do. Else, if you want to be listed as a contributor, follow these steps below:

1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:

YourGitHubName

2.) Then log out and log back into Codecademy using your GitHub login:

ReLogIn

> **Note:**  Your email used for Codecademy will have to be listed in your GitHub profile for the login feature to work properly. If you use the same email for both GitHub and Codecademy already, then you're set.

cc: @yangc95 @SSwiniarski

**Pull request merged**